app/helpers/application_helper.rb

```diff
@@ -32,4 +32,9 @@ def assocation_empty?(assoc)
     assoc.reload unless assoc.nil? || assoc.loaded?
     assoc.nil? ? true : assoc.size == 0
   end
+
+  def i18n_time_ago_in_words(past_time)
+    "#{time_ago_in_words(past_time)} #{t('time_ago')}"
```

Comment:
@thesuss - what I meant was, sometimes if translations (or word sequences) are stuck together (like sticking lego toy pieces together) you can get mangled wording for a particular language.

For instance, here the wording for English would be `3 days ago`.

However, lets say that the equivalent, correct phrase in Swedish would be `past time 3 days` (totally made up, of course), but with lego construction it would be `3 days past time` (and hence incorrect).

In any case, if the correct approach works fine Swedish then we don't have a problem - especially since we don't anticipate supporting other languages.